lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -24,18 +25,17 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
   const { OAuth2RedirectError = /** @type {OAuthErrorConfig} */ ({}) } =
     useContext(Config);
 
-  const {
-    authUrl = null,
-    errorCode = null,
-    errorDetails = '',
-    canvasScopes = /** @type {string[]} */ ([]),
-  } = OAuth2RedirectError ?? {};
+  const { authUrl = null, canvasScopes = /** @type {string[]} */ ([]) } =
+    OAuth2RedirectError ?? {};
 
-  const error = { code: errorCode, details: errorDetails };
+  const error = new AppConfigError({
+    errorCode: OAuth2RedirectError?.errorCode,
+    errorDetails: OAuth2RedirectError.errorDetails,
```

Comment:
The line above suggests that `OAuth2RedirectError` might be nullish, whereas this line assumes it isn't.